automation-tutorial/pom.xml

```diff
@@ -52,6 +52,10 @@
           <artifactId>maven-compiler-plugin</artifactId>
           <version>3.8.0</version>
         </plugin>
+        <plugin>
```

Comment:
I noticed this is a lower version for the `maven-compiler-plugin`. 
We already have version 3.8.0. Will be a good idea to check maven central for the latest version. 

Kindly delete plugin addition, then update the version from 3.8.0 to the latest.

once done, I will merge

Reply:
i've updated the pom.xml file

Reply:
There is still a duplicate plugin. Kindly delete the one with the lower version
```
<plugin>
          <artifactId>maven-compiler-plugin</artifactId>
          <version>3.8.1</version>
        </plugin>
        <plugin>
          <artifactId>maven-compiler-plugin</artifactId>
          <version>3.5.1</version>
        </plugin>
```

Reply:
there is only one see below:
<plugin>
          <artifactId>maven-compiler-plugin</artifactId>
          <version>3.8.1</version>
        </plugin>

I've searched for 3.5.1 but its no longer in the pom file